src/components/shared/wizard/RenderField.tsx

```diff
@@ -229,6 +229,15 @@ const EditableSingleSelect = (props: EditableSingleSelectProps) => {
 		metadataField={metadataField}
 		text={text}
 		form={form}
+		options={
+			metadataField.collection
+				? metadataField.collection.map(item => ({
+					label: item.label ?? item.name,
+					value: item.value,
+					order: item.order,
+				}))
+				: []
+			}
```

Comment:
you can also do
```suggestion
		options={(metadataField.collection ?? []).map(item => ({
			...item,
			label: item.label ?? item.name,
		}))}
```

but it's functionally equivalent, so not super necessary to change.